services/server/server.go

```diff
@@ -596,6 +599,14 @@ func (s *Server) ChangeSettings(ctx context.Context, req *serverpb.ChangeSetting
 		return nil, err
 	}
 
+	if req.EnableStt {
+		// force checks download and execution when STT is enabled
+		err = s.checksService.StartChecks(ctx)
```

Comment:
looks like it's better to run this in a separate goroutine.

Reply:
changed